Sources/KSCrashRecording/include/KSCrash.h

```diff
@@ -231,6 +103,9 @@ typedef enum
 
 #pragma mark - API -
 
+/** Init KSCrash instance with custom base path. */
+- (id) initWithBasePath:(NSString*) basePath;
```

Comment:
I see that you're copying the method from other place, but why it's not `instancetype` here?